Sources/JExtractSwift/ImportedDecls.swift

```diff
@@ -73,6 +74,34 @@ public enum NominalTypeKind {
   case `class`
   case `enum`
   case `struct`
+  case `void`  // TODO: NOT NOMINAL, BUT...
```

Comment:
I'll work on removing this `NominalTypeKind` so we replace it with `SwiftNominalTypeDeclaration.Kind` but we'll have to do a bit of a dance to pull it off -- perhaps make it optional somehow or enum over "nominal or not nominal" etc